<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectWalletPathWidget</class>
 <widget class="QWidget" name="SelectWalletPathWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="pathBtn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>313</y>
     <width>65</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>选择路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathLineEdit">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>310</y>
     <width>285</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>290</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(192, 196, 212);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Select the path of the wallet data</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>188</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome_label">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>190</y>
     <width>121</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_version">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>240</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>v1.0.1</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>332</y>
     <width>360</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
